wiki/Storyboard_Scripting/Cheat_Sheet/en.md

```diff
@@ -41,24 +41,25 @@ Z-order (back to front) is determined by the order the files appear in the .osu
 
 **layer**:
 
--   Background
--   Fail
--   Pass
--   Foreground
+-   0 - Background
+-   1 - Fail
+-   2 - Pass
+-   3 - Foreground
 
 *Note: Background and Foreground are always visible.*
 
 **origin**:
 
--   TopLeft
--   TopCentre
--   TopRight
--   CentreLeft
--   Centre
--   CentreRight
--   BottomLeft
--   BottomCentre
--   BottomRight
+-   0 - TopLeft
```

Comment:
I would just add a note here that 6 can also be TopLeft, but you shouldn't use it. Custom is not used for storyboarding

Reply:
doesn't have to be confusing, just note that it does the same thing as topleft. if a storyboarder did use it for some reason, other people might want to know what 6/Custom means

Reply:
okay, readded them